index.js

```diff
@@ -253,6 +262,7 @@ SassCompiler.prototype.brunchPlugin = true;
 SassCompiler.prototype.type = 'stylesheet';
 SassCompiler.prototype.pattern = /\.s[ac]ss$/;
 SassCompiler.prototype._bin = isWindows ? 'sass.bat' : 'sass';
-SassCompiler.prototype._compass_bin = isWindows ? 'compass.bat' : 'compass'; //eslint-disable-line camelcase
+// eslint-disable-next-line camelcase
```

Comment:
Maybe we should disable camelcase for the whole file? `eslint` comments in code look shitty.